guide/java/policies.md

```diff
@@ -31,42 +31,144 @@ Policies are highly reusable as their inputs, thresholds and targets are customi
 
 ### Management Policies
 
-- AutoScaler Policy
+#### AutoScaler Policy
    
-   Increases or decreases the size of a Resizable entity based on an aggregate sensor value, the current size of the entity, and customized high/low watermarks.
+Increases or decreases the size of a Resizable entity based on an aggregate sensor value, the current size of the entity, and customized high/low watermarks.
 
-   An AutoScaler policy can take any sensor as a metric, have its watermarks tuned live, and target any resizable entity - be it an application server managing how many instances it handles, or a tier managing global capacity.
+An AutoScaler policy can take any sensor as a metric, have its watermarks tuned live, and target any resizable entity - be it an application server managing how many instances it handles, or a tier managing global capacity.
 
-   e.g. if the average request per second across a cluster of Tomcat servers goes over the high watermark, it will resize the cluster to bring the average back to within the watermarks.
-  
-<!---
-TODO - list some
-TODO - describe how they can be customised (briefly mention sensors)
--->
+e.g. if the average request per second across a cluster of Tomcat servers goes over the high watermark, it will resize the cluster to bring the average back to within the watermarks.
+
+{% highlight yaml %}
+
+  - type: org.apache.brooklyn.policy.autoscaling.AutoScalerPolicy
```

Comment:
Perhaps show a `brooklyn.policies:` before this type, to make it clearer how/where this is added?